packages/database/src/migrations/20220909021244-AddAnalyticsDataTable-modifies-data.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  return insertObject(db, 'data_table', {
+    id: generateId(),
+    code: 'analytics',
+    description: 'Fetches data for individual data elements',
+    type: 'internal',
+    permission_groups: '{*}',
```

Comment:
analytics is available to all, since data-broker will handle permissions checks to ensure users can only pull data they have access to